<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>man_hinh_soan_cau_hoi</class>
 <widget class="QWidget" name="man_hinh_soan_cau_hoi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 5px solid rgb(0, 112, 192);
border-radius: 10px</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>201</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: None;
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>207</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: None;
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: None;</string>
      </property>
      <property name="text">
       <string>Môn học</string>
      </property>
     </widget>
     <widget class="QComboBox" name="mon_hoc">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 3px solid rgba(46, 82, 101, 200);
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: None;</string>
      </property>
      <property name="text">
       <string>Độ khó</string>
      </property>
     </widget>
     <widget class="QComboBox" name="do_kho">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 3px solid rgba(46, 82, 101, 200);
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="thoat">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 3px solid rgb(0, 0, 0);
background-color: rgb(85, 170, 255);
border-radius: 10px</string>
     </property>
     <property name="text">
      <string>Thoát</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/images/32_n.png</normalon>
      </iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>0</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../resource.qrc">:/images/trang chủ hình nón.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="them">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 3px solid rgb(0, 0, 0);
background-color: rgb(85, 170, 255);
border-radius: 10px

</string>
     </property>
     <property name="text">
      <string>Thêm</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/images/6_n.png</normalon>
      </iconset>
     </property>
    </widget>
    <zorder>groupBox_11</zorder>
    <zorder>label</zorder>
    <zorder>thoat</zorder>
    <zorder>them</zorder>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>691</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 5px solid rgb(0, 112, 192);
border-radius: 10px</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTextEdit" name="cau_hoi">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>611</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 4px solid rgba(46, 82, 101, 200);

border-radius: 10px </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="cau_tra_loi">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>611</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 4px solid rgba(46, 82, 101, 200);

border-radius: 10px</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: None;</string>
    </property>
    <property name="text">
     <string>Câu hỏi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: None;</string>
    </property>
    <property name="text">
     <string>Câu trả lời</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
